test/fuzzing/build.sh

```diff
@@ -0,0 +1,28 @@
+export FUZZER_DIR=$SRC/linkerd2/test/fuzzing
+
+mkdir $SRC/linkerd2/test/fuzzing
```

Comment:
If `$SRC` contains whitespaces, this line and a lot of other lines in this script will fail.